pkgs/by-name/nu/numbat/package.nix

```diff
@@ -5,20 +5,28 @@
 , rustPlatform
 , darwin
 , numbat
+, tzdata
 }:
 
 rustPlatform.buildRustPackage rec {
   pname = "numbat";
-  version = "1.12.0";
+  version = "1.13.0";
 
   src = fetchFromGitHub {
     owner = "sharkdp";
     repo = "numbat";
     rev = "v${version}";
-    hash = "sha256-MYoNziQiyppftLPNM8cqEuNwUA4KCmtotQqDhgyef1E=";
+    hash = "sha256-o3EYhMFBgs/Ni+YCM3+RdUYlwRt+nMaEP/cAkDXMVHc=";
   };
 
-  cargoHash = "sha256-t6vxJ0UIQJILCGv4PO5V4/QF5de/wtMQDkb8gPtE70E=";
+  cargoHash = "sha256-rK9RPd/hww2F87l/dd14pB4izE58NuqaewYaqMimV1M=";
+
+  patches = [
+    # Evaluating the currencies module (caused by referencing any non-euro Money
+    # unit) requires network access. See sharkdp/numbat#560.
+    # Remove this when updating to 1.14.0.
+    ./test_xkcd_2812.patch
```

Comment:
Wouldn't the upstream patch be preferable?

https://github.com/sharkdp/numbat/commit/272c2455136b6e8debc3003b22db18fba6442822.patch

Reply:
Good catch! Yes, I would definitely suggest taking the patch from upstream.

Reply:
I did request the change but didn't think about using it ahead of time. I'll fix that up.

Reply:
https://github.com/sharkdp/numbat/commit/4756a1989ecdab35fd05ca18c721ed15d8cde2b1

Sorry pasted the wrong commit, this one is the one that got merged upstream